apps/src/templates/progress/StageTrophyProgressBubble.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component, PropTypes } from 'react';
+import color from "@cdo/apps/util/color";
+import trophy from '!!file-loader!./images/trophy.svg';
```

Comment:
Why can't we use the url-loader we use by default for other SVGs?

Reply:
I was having trouble getting the base64-encoded svg to play nice with the `<use href=...>` tag.